addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -469,5 +298,9 @@ registerModel({
             readonly: true,
             required: true,
         }),
+        view: one('DiscussView', {
```

Comment:
Until we have domain in name I would call it `discussView` for consistency and ease of maintenance.